ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonVnextDemoController.swift

```diff
@@ -16,27 +16,36 @@ class ButtonVnextDemoController: DemoController {
             for style in MSFButtonVnextStyle.allCases {
                 addDescription(text: "\(style.description) style:", textAlignment: .natural)
 
-                let button = MSFButtonVnext(style: style, size: size) {
-                    self.didPressButton()
-                }
+                let button = MSFButtonVnext(style: style, size: size, action: { [weak self] _ in
+                    guard let strongSelf = self else {
+                        return
+                    }
+
+                    strongSelf.didPressButton()
+                })
                 button.state.text = "Button"
 
-                let disabledButton = MSFButtonVnext(style: style, size: size) {
-                    self.didPressButton()
-                }
+                let disabledButton = MSFButtonVnext(style: style, size: size, action: { [weak self] _ in
+                    guard let strongSelf = self else {
+                        return
+                        
+                    }
+
+                    strongSelf.didPressButton()
+                })
                 disabledButton.state.text = "Button"
                 disabledButton.state.isDisabled = true
 
                 addRow(items: [button.view, disabledButton.view], itemSpacing: 20)
 
                 if let image = style.image {
-                    let iconButton = MSFButtonVnext(style: style, size: size) {
+                    let iconButton = MSFButtonVnext(style: style, size: size) {_ in
```

Comment:
A optional convenience callback without sender would be good addition to reduce the unnecessary parameters

Reply:
Closures (or blocks in objC) have been preferred over selectors. I believe we should not support selectors.
Let's talk about the callback with no parameters offline.